pkg/kotsadm/objects/minio_objects.go

```diff
@@ -222,7 +222,7 @@ func migrateToMinioXlInitContainers(deployOptions types.DeployOptions, resourceR
 
 	return []corev1.Container{
 		{
-			Image:           GetAdminConsoleImage(deployOptions, "minio-client"),
+			Image:           GetAdminConsoleImage(deployOptions, "minio"),
```

Comment:
Previously we needed `minio/mc` image to copy the `mc` binary for use in later initContainers. Newer `minio/minio` images now include the `mc` binary at `/usr/bin/mc`, so we can just use that and don't need to ship the additional `minio/mc` image.